src/ashell/shell-state.c

```diff
@@ -88,13 +88,26 @@ int32_t ashell_get_filename_buffer(const char *buf, char *destination)
 
     if (arg_len == 0) {
         *destination = '\0';
-        comms_printf(ERROR_NOT_ENOUGH_ARGUMENTS);
+        comms_println(ERROR_NOT_ENOUGH_ARGUMENTS);
         return RET_ERROR;
     }
 
     return arg_len;
 }
 
+static int8_t file_exists(const char *buf, char *filename)
```

Comment:
This function name isn't great because it also gets the filename. That got me looking into it more.

I think a better way to fix this is to change fs_open_alloc() in file_utils.c. It takes a `mode` argument but mostly ignores it. If it starts with 'r' it should not be creating a new file. That would solve the problem I think.